Kommunicate-io/Kommunicate-Web-SDK
# Test to send emoji, file and location

### What do you want to achieve?
-Added test case for sending emoji, files, and location
-Fixed Locator values

### PR Checklist


### How was the code tested?

- navigate to cd `playwright_test/`
- run `npx playwright test`
*Note* 
- To test code inside the tests folder make sure you have updated the file dir line 79 `chatOptions.spec.js`


### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch
